packages/menu/test/menu-item.test.ts

```diff
@@ -169,6 +169,24 @@ describe('Menu item', () => {
         expect(el.value).to.equal('Selected Text');
         expect(el.hasAttribute('value')).to.be.false;
     });
+    it('renders description', async () => {
+        const el = await fixture<MenuItem>(
+            html`
+                <sp-menu-item selected>
+                    Menu Item Text
+                    <span slot="description">
+                        Description for the Menu-Item
+                    </span>
+                </sp-menu-item>
+            `
+        );
+
+        const descriptionSlot = el.shadowRoot.querySelector(
+            'slot[name="description"]'
+        );
+
+        expect(descriptionSlot).to.not.be.null;
```

Comment:
Testing wise, this tests that the code you wrote above went unchanged when rendered to the page. In the case that you are testing a renderer, great, in the case that you're testing a custom element, maybe not the most productive. The assertion to guarantee the contract here is more likely:
```suggestion
        expect(descriptionElement.assignedSlot).to.not.be.null;
```
I might change the title of the test to `'assigns content to the "description" slot'`, and change the name of the variable used to `descriptionElement` as the "description slot" is the `<slot name="description">` element within the shadow DOM of the Menu Item.